app/assets/stylesheets/search.scss

```diff
@@ -90,23 +97,55 @@
 }
 
 .my-flight {
-  border-bottom: 1px solid $my-gray;
+  border: 1px solid $light-gray;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
   font-size: 0.85em;
-  height: 4em;
-  padding: 0;
+  vertical-align: center;
+  p{
+    text-align: center;
+  }
 }
-
 .all-flight {
   color: $my-black;
   font-size: 1em;
   font-weight: bold;
+  text-align: center;
 
 }
-
 .box {
-  border-radius: 10px;
-  padding-left: 1em;
+  background-color: $my-white;
+  border-radius: 4px;
+  padding:  1em;
   position: relative;
+  width: 90% !important;
```

Comment:
Properties should be ordered background-color, border-radius, padding, position, top, width